presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/SymbolAliases.java

```diff
@@ -93,7 +94,9 @@ private static String toKey(String alias)
         ImmutableMap.Builder<String, SymbolReference> mapUpdate = ImmutableMap.builder();
         for (Map.Entry<Symbol, Expression> assignment : assignments.getMap().entrySet()) {
             for (Map.Entry<String, SymbolReference> existingAlias : map.entrySet()) {
-                if (assignment.getValue().equals(existingAlias.getValue())) {
+                if (assignment.getValue().equals(existingAlias.getValue())
```

Comment:
I don't think this is the way to go. I would prefer to use `CAST` explicitly in tests and not to have it magically ignore `CAST`.

For example how would you test that no `CAST` is added?